satpy/writers/ninjotiff.py

```diff
@@ -49,5 +49,5 @@ def save_image(self, img, filename=None, **kwargs):  # floating_point=False,
 
         .. _ninjotiff: http://www.ssec.wisc.edu/~davidh/polar2grid/misc/NinJo_Satellite_Import_Formats.html
         """
-        filename = filename or self.get_filename(**img.info)
+        filename = filename or self.get_filename()
```

Comment:
Shouldn't this be `**img.attrs`?

Reply:
Sorry:

```
filename = filename or self.get_filename(**img.data.attrs)
```

Reply:
Yes, you are right, I have just tested it. I update this PR right away, thanks for checking.